run_jobs.py

```diff
@@ -118,15 +119,26 @@ def run_jobs(self) -> None:
 
 
 if __name__ == '__main__':
+    db_creator_obj = DBCreator(ENV['INVENTORY_DB'], ENV['APPEND_TABLE_NAMES'])
     how_started = os.environ.get('HOW_STARTED', None)
 
     if how_started == 'DOCKER_COMPOSE':
-        logger.info('Waiting for the MySQL turtle, hehe')
         # Wait for MySQL container to finish setting up
-        time.sleep(30.0)
+        logger.info('Waiting for the MySQL turtle')
+        waiting = True
+        while waiting:
+            time.sleep(3.0)
+            try:
+                db_creator_obj.set_up()
+                db_creator_obj.tear_down()
+                logger.info('MySQL caught up')
+                waiting = False
+            except sqlalchemy.exc.OperationalError:
+                logger.debug('Still waiting!')
```

Comment:
See 5:17 in https://www.youtube.com/watch?v=gyDb4szpWmc.

Reply:
We should make it a requirement for linking to YouTube in code reviews -:)

Maybe this PR will fix my problem of not able to connect to MySQL instance with docker-compose approach. I used virtualenv afterwards.

Reply:
I always test my DB changes in docker-compose and I did not have the problem of connecting to DB, deleting and inserting records

Reply:
Yeah, I'm happy to help you @zqian figure out any issues with your `docker-compose` setup. This was just to make it so you don't have to wait 30 seconds every time on start up.

Reply:
> See 5:17 in https://www.youtube.com/watch?v=gyDb4szpWmc.

![sameAsItEverWas](https://user-images.githubusercontent.com/20786/78995710-fea2d980-7b10-11ea-94ca-1e9ed5ca4edd.gif)

Qu'est-ce que c'est?

PS: YouTube URLs can include the start time: https://youtu.be/gyDb4szpWmc?t=317